src/mintpy/objects/euler_pole.py

```diff
@@ -539,169 +542,203 @@ def transform_xyz_enu(lat, lon, x=None, y=None, z=None, e=None, n=None, u=None):
 # check usage: https://github.com/yuankailiu/utils/blob/main/notebooks/PMM_plot.ipynb
 # Later will be moved to a separate script `plot_utils.py` in plate motion package
 
-def read_plate_outlines(filename, order='lola'):
-    """Load the plate boundary text files
-    Paramters:
-        filename - filename of the boundary text file
-        order    - the order of columns, 'lalo' or 'lola', default: (lola)
-    Returns:
-        Bnds     - a dictionary that contains a list of vertices of the plate polygon (lat, lon)
+def read_plate_outline(pmm_name='GSRM'):
+    """Read the plate boundaries for the given plate motion model.
+
+    Paramters: pmm_name - str, plate motion (model) name
+    Returns:   outlines - dict, a dictionary that contains a list of vertices of the plate polygon (lat, lon)
     """
-    datatype = filename.split('data/plate_boundary/')[-1].split('/')[0]
-    pDict = {}
-    if 'GSRM' in datatype:
-        for key, val in GSRM_V21_PMM.items():
-            pDict[val.Abbrev.upper()] = key
-    elif 'MORVEL56' in datatype:
-        for key, val in NNR_MORVEL56_PMM.items():
-            if val.Abbrev == 'nb': val.Abbrev = 'nu'    # replace duplicate capitalized abbv for 'Nubia'
-            pDict[val.Abbrev.upper()] = key
-    Bnds = {}
-    with open(filename) as f:
+
+    # check input
+    if 'GSRM' in pmm_name:
+        pmm_name = 'GSRM'
+        pmm_dict = GSRM_V21_PMM
+        coord_order = 'lola'
+
+    elif 'MORVEL' in pmm_name:
+        pmm_name = 'MORVEL'
+        pmm_dict = NNR_MORVEL56_PMM
+        coord_order = 'lalo'
```

Comment:
This place should be 'lola' as well.

Because I also flip the order in the file from the original `All_boundary` in MORVEL_56.

So we can just set the default as `coord_order='lola'` and keep the file as it is (and it also makes sense since it has `.gmt` extension.).

But this way, we no longer keep the original downloaded file untouched. But anyway, I already added those `> ` signs in the file.

Reply:
As we discussed offline:
+ let's keep the original downloaded file content untouched, for reproducibility. 
+ use `.lonlat` and `.latlon` as file extensions, instead of `.gmt`, to be more informative.
+ update the `data/plate_boundary/README.md` file to be consistent with the above changes.